frontend/src/scenes/ingestion/panels/PlatformPanel.tsx

```diff
@@ -10,7 +10,7 @@ export function PlatformPanel(): JSX.Element {
     const { setPlatform } = useActions(ingestionLogic)
 
     return (
-        <div className="welcome-panel">
+        <div style={{ maxWidth: 400 }}>
```

Comment:
👀

Reply:
This is one of those cases where I think style makes sense as in this case we only want to limit the modal width for these specific views. I could however get rid of the lint error to at least indicate it is on purpose 👍